pkg/apiserver/registry/inventory/vpc_suite_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package inventory
+package inventory_test
```

Comment:
Dont change package name.. keep it inventory only

Reply:
solved